templates/oned.conf.erb

```diff
@@ -72,6 +75,10 @@ SCRIPTS_REMOTE_DIR=/var/tmp/one
 
 PORT = <%= scope.lookupvar('one::oned_port') %>
 
+<% if scope.function_versioncmp([@one_version,'4.14']) >= 0 -%>
```

Comment:
Can you try to keep the variable lookup in the same pattern:
scope.lookupvar('...')
This prevents the dynamic variable lookup from templates which might occur in Puppet 3 and which will stop working in Puppet 4.